test/source/TestUtility.cpp

```diff
@@ -543,6 +543,23 @@ static int TestUtilityExchange()
 		EATEST_VERIFY(a == "EA Standard Template Library");
 	}
 
+	// Issue #82; construct pair using single move constructor
+	#if EASTL_MOVE_SEMANTICS_ENABLED
+	{
+		class TestPairSingleMoveConstructor
```

Comment:
Fix the intend formatting here.  Since you are changing might as well make it a struct and remove the public.  Left justify the '&&'.  You can can also just use clang-format as EASTL has its format script in the package root.